build.sbt

```diff
@@ -72,6 +73,8 @@ Revolver.settings
 
 Revolver.enableDebugging(port = 5050, suspend = false)
 
+net.virtualvoid.sbt.graph.Plugin.graphSettings
```

Comment:
this and the new plugin below allows visualization of the dependency tree with
sbt dependency-tree